### PR TITLE
Update docs and code to point to new repo url.

### PR DESCRIPTION
**Pull request by akashnimare**
---

**What's this PR do?**

**Any background context you want to provide?**

**Screenshots?**

**You have tested this PR on:**
  - [ ] Windows
  - [ ] Linux/Ubuntu
  - [ ] macOS

### CODE REVIEW

File: development.md [hunk 1]
```diff
@@ -105,16 +105,16 @@ Now you are ready for next step [: Get Zulip Electron Code.](https://github.com/
...
 
 
 # Get Zulip Electron Code
 
-1. In your browser, visit https://github.com/zulip/zulip-electron and click the `fork` button. You will need to be logged in to GitHub to do this.
+1. In your browser, visit https://github.com/zulip/zulip-desktop and click the `fork` button. You will need to be logged in to GitHub to do this.
 2. Open Terminal (macOS/Ubuntu) or Git BASH (Windows; must **run as an Administrator**).
-3. In Terminal/Git BASH, [clone your fork of the zulip-electron repository](https://github.com/zulip/zulip-electron/blob/master/development.md#clone-to-your-machine) and [connect the zulip-electron upstream repository](https://github.com/zulip/zulip-electron/blob/master/development.md#connect-your-fork-to-zulip-electron-upstream)
+3. In Terminal/Git BASH, [clone your fork of the zulip-desktop repository](https://github.com/zulip/zulip-desktop/blob/master/development.md#clone-to-your-machine) and [connect the zulip-deskktop upstream repository](https://github.com/zulip/zulip-desktop/blob/master/development.md#connect-your-fork-to-zulip-desktop-upstream)
```
Comment (kanishk98) [inline development.md]: Typo here, there should only be one k in desktop.

File: development.md [hunk 2]
```diff
@@ -105,16 +105,16 @@ Now you are ready for next step [: Get Zulip Electron Code.](https://github.com/
     C:\Windows\system32> npm install --global --production windows-build-tools
 
 
-**if** [**NPM**](https://www.npmjs.com/get-npm) **and** [**node-gyp**](https://github.com/nodejs/node-gyp#installation) **don't come bundled with your Node.js installation, Download manually** 
+**if** [**NPM**](https://www.npmjs.com/get-npm) **and** [**node-gyp**](https://github.com/nodejs/node-gyp#installation) **don't come bundled with your Node.js installation, Download manually**
 
-Now you are ready for next step [: Get Zulip Electron Code.](https://github.com/zulip/zulip-electron/blob/master/development.md#get-zulip-electron-code)
+Now you are ready for next step [: Get Zulip Electron Code.](https://github.com/zulip/zulip-desktop/blob/master/development.md#get-zulip-desktop-code)
```
Comment (kanishk98) [inline development.md]: Should we change the name to Zulip Desktop here?

**Review by kanishk98 — COMMENTED** [review]
Looks good to me for the most part. I found a few files that still need to be updated with the correct URL though:

1. `app/renderer/js/notification/helpers.js`
2. `tests/e2e/package.json`
3. `package.json`, the `appId` value.

**Comment by akashnimare** [comment]
We can leave appId as it is.

**Pull request merged**